textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/predicates/SelectPredicate.java

```diff
@@ -5,8 +5,8 @@
 /**
  * Object representation of a "SELECT (...)" predicate inside a { @code SelectExtractStatement }.
  * Subclasses have specific fields related to its projection functionalities.
- * SelectPredicate --+ SelectAllPredicate
- *                   + SelectFieldsPredicate
+ * SelectPredicate --+ SelectAllFieldsPredicate
```

Comment:
We can have the "field" vs "attribute" discussion here.

In the current codebase, "attribute" is often used associated with the schema. Say a table's schema contains 3 "attributes". "field" is used associated with a "cell" inside a tuple. Say the tuple has a "TextField", a "StringField". We don't say a "StringAttribute".

Currently, the confusion of "attribute" and "field" comes from the "attribute" class itself. The attribute class consists of two variables, "fieldName" and "fieldType". So we can see something like `attributeName = attribute.getFieldName()` in our codebase. And a lot of places in our code base uses the terms "attributeList" and "attributeNames"

I don't have any opinion on this right now. Any thoughts @FlavioBayer @chenlica ?